### PR TITLE
Add phpstan

### PR DESCRIPTION
**Pull request by PrinsFrank**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parameters:
+    level: 9
```
Comment (szepeviktor) [inline phpstan.neon:2]: @PrinsFrank That is brave 🏋🏻‍♂️

**Comment by szepeviktor** [comment]
#23 is sent to this branch.

**Comment by szepeviktor** [comment]
Before merging please merge my PR.

File: .gitattributes [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 # Ignore all test and documentation with "export-ignore".
+/.github            export-ignore
+/dev                export-ignore
 /docs               export-ignore
 /tests              export-ignore
 /.gitattributes     export-ignore
 /.gitignore         export-ignore
-/.php-cs-fixer      export-ignore
+/.php-cs-fixer.php  export-ignore
+/phpstan.neon
```
Comment (szepeviktor) [inline .gitattributes:12]:
```suggestion
/phpstan.neon       export-ignore
```

**Comment by PrinsFrank** [comment]
> Before merging this please merge my PR.

I will look at this tomorrow, I want to do some performance testing before merging as I've not seen this method of getting enum cases before. ATM looking at the livestream for the first JWT image, so signing off today. Thanks for all the work today!

**Pull request merged**